.github/workflows/integration-tests.yml

```diff
@@ -14,43 +14,4 @@ env:
   APP_ID: 1
 
 jobs:
```

Comment:
If you want to move everything to the same workflow this file should be deleted

Reply:
yes, I never got to continue working on this one, and Im afraid that I won't for a few days 
If you get to look to the error with travis, we can rescue this PR...

Reply:
Removed